pkg/operator/controller/certificate/default_cert.go

```diff
@@ -43,12 +43,16 @@ func (r *reconciler) ensureDefaultCertificateForIngress(caSecret *corev1.Secret,
 	case desired == nil && current == nil:
 		// Nothing to do.
 	case desired == nil && current != nil:
-		if err := r.deleteRouterDefaultCertificate(current); err != nil {
+		if deleted, err := r.deleteRouterDefaultCertificate(current); err != nil {
 			return fmt.Errorf("failed to delete default certificate: %v", err)
+		} else if deleted {
+			r.recorder.Eventf(ci, "Normal", "DeletedDefaultCertificate", "Deleted default wildcard certificate %q", desired.Name)
 		}
 	case desired != nil && current == nil:
-		if err := r.createRouterDefaultCertificate(desired); err != nil {
+		if created, err := r.createRouterDefaultCertificate(desired); err != nil {
 			return fmt.Errorf("failed to create default certificate: %v", err)
+		} else if created {
+			r.recorder.Eventf(ci, "Normal", "CreatedDefaultCertificate", "Created default wildcard certificate %q", desired.Name)
```

Comment:
What's the guideline for using events over logging?

Reply:
Up to us to decide what's most user friendly AFAIK